src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -51,7 +51,14 @@ export default function applicantInformation(schema, options) {
       'ui:order': fields,
       relativeFullName: fullName.uiSchema,
       relativeSocialSecurityNumber: ssn.uiSchema,
-      relativeDateOfBirth: currentOrPastDate.uiSchema('Date of birth'),
+      relativeDateOfBirth: _.assign(currentOrPastDate.uiSchema('Date of birth'),
+        {
+          'ui:errorMessages': {
+            pattern: 'Please provide a valid date',
```

Comment:
Why does this particular date need these validation messages, vs what's there by default? Should we make the changes for all current or past dates?